src/pages/Applications/List/Add/Form.tsx

```diff
@@ -1,88 +1,78 @@
 import { FieldValues, useForm } from "react-hook-form";
-import InputWithLabel from "../../components/InputWithLabel";
+import InputWithLabel from "../../../../components/InputWithLabel";
 import { zodResolver } from "@hookform/resolvers/zod";
-import { formatDate, formSchema } from "./addFormSchema";
+import { defaultFormValues, formSchema } from "./formSchema";
 
-function AddForm() {
+export default function Form() {
```

Comment:
Add feature can be moved to Application folder

Reply:
done